From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 24 Dec 2020 18:18:54 +0100
Subject: Examples: use python3 interpreter instead of python

Forwarded: https://github.com/streamlink/streamlink/pull/3445
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 examples/gst-player.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/gst-player.py b/examples/gst-player.py
index 79a5fdd..e26551f 100755
--- a/examples/gst-player.py
+++ b/examples/gst-player.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
