From: Alexis Murzeau <amubtdx@gmail.com>
Date: Fri, 23 Aug 2019 19:18:35 +0200
Subject: plugins.pixiv: fix doc typo thats -> that's

Forwarded: https://github.com/streamlink/streamlink/pull/2600
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 src/streamlink/plugins/pixiv.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/streamlink/plugins/pixiv.py b/src/streamlink/plugins/pixiv.py
index 7cdee83..75d9aa4 100644
--- a/src/streamlink/plugins/pixiv.py
+++ b/src/streamlink/plugins/pixiv.py
@@ -79,7 +79,7 @@ class Pixiv(Plugin):
             sensitive=True,
             metavar="SESSIONID",
             help="""
-        The pixiv.net sessionid thats used in pixivs PHPSESSID cookie.
+        The pixiv.net sessionid that's used in pixivs PHPSESSID cookie.
         can be used instead of the username/password login process.
         """
         ),
@@ -88,7 +88,7 @@ class Pixiv(Plugin):
             sensitive=True,
             metavar="DEVICETOKEN",
             help="""
-        The pixiv.net device token thats used in  pixivs device_token cookie.
+        The pixiv.net device token that's used in pixivs device_token cookie.
         can be used instead of the username/password login process.
         """
         ),
