From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 16 Jul 2022 16:35:11 +0200
Subject: test_dash.py: fix importing MonkeyPatch on old pytest

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 tests/stream/test_dash.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/tests/stream/test_dash.py b/tests/stream/test_dash.py
index 49dfa4c..78b09f3 100644
--- a/tests/stream/test_dash.py
+++ b/tests/stream/test_dash.py
@@ -3,6 +3,7 @@ from typing import List
 from unittest.mock import ANY, MagicMock, Mock, call, patch
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
 
 from streamlink import PluginError
 from streamlink.stream.dash import DASHStream, DASHStreamWorker
@@ -305,13 +306,13 @@ class TestDASHStream(unittest.TestCase):
 
 class TestDASHStreamWorker:
     @pytest.fixture
-    def mock_time(self, monkeypatch: pytest.MonkeyPatch) -> Mock:
+    def mock_time(self, monkeypatch: MonkeyPatch) -> Mock:
         mock = Mock(return_value=1)
         monkeypatch.setattr("streamlink.stream.dash.time", mock)
         return mock
 
     @pytest.fixture(autouse=True)
-    def mock_wait(self, monkeypatch: pytest.MonkeyPatch) -> Mock:
+    def mock_wait(self, monkeypatch: MonkeyPatch) -> Mock:
         mock = Mock(return_value=True)
         monkeypatch.setattr("streamlink.stream.dash.DASHStreamWorker.wait", mock)
         return mock
@@ -355,7 +356,7 @@ class TestDASHStreamWorker:
 
     def test_dynamic_reload(
         self,
-        monkeypatch: pytest.MonkeyPatch,
+        monkeypatch: MonkeyPatch,
         worker: DASHStreamWorker,
         representation: Mock,
         segments: List[Mock],
